src/components/JobLogs/withJobLogsSort.js

```diff
@@ -138,7 +132,7 @@ const withJobLogsSort = WrappedComponent => {
     };
 
     render() {
-      const hasLoaded = get(this.context, ['jobs', 'hasLoaded'], false);
+      const { hasLoaded = false } = this.context;
```

Comment:
Is default value needed? Why it cannot be retrieved right away from context as is?